src/x86-64.S

```diff
@@ -1651,9 +1651,15 @@ DES_bs_crypt_plain_loop:
 #define CX_XOP                  $0x00000800
 
 /* Leaf 7 */
+// 	0x00010000	/* AVX512 foundation */
+// 	0x00020000	/* AVX512 Doubleword and Quadword Instructions */
+// 	0x40000000	/* AVX512BW */
+// 	0x80000000	/* AVX512VL */
 #define C7_AVX2                 $0x00000020 /* AVX2 */
 #define C7_AVX512F              $0x00010000
 #define C7_AVX512BW             $0x40010000 /* AVX512BW + AVX512F */
+#define C7_AVX512VL             $0xC0010000 /* AVX512BW + AVX512VL + AVX512F */
+#define C7_AVX512DQ             $0xC0030000 /* AVX512BW + AVX512DQ + AVX512VL + AVX512F */
```

Comment:
(I didn't review the specific bitmasks against the documentation. I just hope they're correct.)